stdlib/math.jou

```diff
@@ -111,110 +155,88 @@ declare nan(tagp: byte*) -> double
 
 # Hyperbolic versions of the trig functions
 @public
-declare cosh(x: double) -> double
-@public
-declare coshf(x: float) -> float
-@public
 declare sinh(x: double) -> double
 @public
 declare sinhf(x: float) -> float
 @public
-declare tanh(x: double) -> double
+declare cosh(x: double) -> double
 @public
-declare tanhf(x: float) -> float
+declare coshf(x: float) -> float
 @public
-declare acosh(x: double) -> double
+declare tanh(x: double) -> double
 @public
-declare acoshf(x: float) -> float
+declare tanhf(x: float) -> float
 @public
 declare asinh(x: double) -> double
 @public
 declare asinhf(x: float) -> float
 @public
+declare acosh(x: double) -> double
+@public
+declare acoshf(x: float) -> float
+@public
 declare atanh(x: double) -> double
 @public
 declare atanhf(x: float) -> float
 
-# Exponential and logarithmic functions
-
-# Computes e^x.
+# Computes 2^x (2 to the power of x).
 @public
-declare exp(x: double) -> double
+declare exp2(x: double) -> double
 @public
-declare expf(x: float) -> float
+declare exp2f(x: float) -> float
 
-# Computes 2^x.
+# Computes e^x (e to the power of x).
 @public
-declare exp2(x: double) -> double
+declare exp(x: double) -> double
 @public
-declare exp2f(x: float) -> float
+declare expf(x: float) -> float
 
-# Computes 10^x.
+# Computes 10^x (10 to the power of x).
 @public
 declare exp10(x: double) -> double
 @public
 declare exp10f(x: float) -> float
 
-# Returns the natural (base e) logarithm of x.
+# Computes x^y (x to the power of y).
 @public
-declare log(x: double) -> double
+declare pow(x: double, y: double) -> double
 @public
-declare logf(x: float) -> float
+declare powf(x: float, y: float) -> float
 
 # Returns the base 2 logarithm of x.
 @public
 declare log2(x: double) -> double
 @public
 declare log2f(x: float) -> float
 
+# Returns the natural (base e) logarithm of x.
+# To get a logarithm with some other base `b`, use `log(x) / log(b)`.
+@public
+declare log(x: double) -> double
+@public
+declare logf(x: float) -> float
+
 # Returns the base 10 logarithm of x.
 @public
 declare log10(x: double) -> double
 @public
 declare log10f(x: float) -> float
 
-# Power functions
-
-# Raise to power.
-@public
-declare pow(x: double, y: double) -> double
-@public
-declare powf(x: float, y: float) -> float
 # Returns the square root of x.
 @public
 declare sqrt(x: double) -> double
 @public
 declare sqrtf(x: float) -> float
+
 # Returns the cubic root of x.
 @public
 declare cbrt(x: double) -> double
 @public
 declare cbrtf(x: float) -> float
+
 # Returns the hypotenuse of a right-angled triangle whose legs are x and y.
+# In other words, return the distance between (0, 0) and (x, y).
 @public
 declare hypot(x: double, y: double) -> double
 @public
 declare hypotf(x: float, y: float) -> float
-
-# Error and gamma functions
-
-# Returns the error function value for x.
-@public
-declare erf(x: double) -> double
```

Comment:
I think I should add back `erf` and `erff`. Without these, there's no good way to compute the integral of `exp(-x^2)`, which is needed for many practical probability calculations.

Basically anything that is a sum or average of many similar things is gaussian distributed. And to work out the probability of the thing being less than or greater than a given value, you need these functions.